src/releng_channel_status/releng_channel_status/templates/layout.html

```diff
@@ -0,0 +1,13 @@
+<html>
+    <head>
+        <title>
+            PAGE TITLE
+        </title>
+        <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-beta/css/bootstrap.min.css" integrity="sha384-/Y6pD6FV/Vv2HJnA6t+vslU6fwYXjCFtcEpHbNJ0lyAFsXTsjBbfaDjzALeQsN6M" crossorigin="anonymous">
```

Comment:
I don't know what's our policy about external CDNs. Using them may be seen as tracking. Maybe @garbas or @mozbhearsum would know.

Reply:
Same as above.

Reply:
@JohanLorenzo good catch. 

Lets first get the functionality working. Once we have everything in place we can tighten the security using [flask-talisman](https://github.com/GoogleCloudPlatform/flask-talisman) which is already used in other projects in ``mozilla-releng/services``.